libs/shared/ui/chart/src/lib/chart/chart.component.html

```diff
@@ -1,5 +1,5 @@
 <google-chart
-  *ngIf="data"
+  *ngIf="chartData"
```

Comment:
there was no variable data defined hence the charts was not even getting displayed. Data was coming in property chartData